scripts/aqm/stats/exevs_aqm_grid2obs_stats.sh

```diff
@@ -221,7 +228,14 @@ if [ -s ${check_file} ]; then
     obs_daily_found=1
 else
     echo "Can not find pre-processed obs daily input ${check_file}"
-    ## add email function here
+    if [ $SENDMAIL = "YES" ]; then
+      export subject=AQM Daily Observed Missing for EVS ${COMPONENT}"
```

Comment:
Add an opening ". The line should be: `export subject="AQM Hourly Observed Missing for EVS ${COMPONENT}"`

Reply:
Fixed.